src/deploy-cromwell-on-azure/scripts/mount.blobfuse

```diff
@@ -14,23 +20,22 @@ function write_log() {
 
 account_name=$(echo $3 | grep -Po '(?i)account_name=\K([^,]*)')
 container_name=$(echo $3 | grep -Po '(?i)container_name=\K([^,]*)')
-account_key=$(echo $3 | grep -Po '(?i)account_key=\K([^,]*)')
-sas_token=$(echo $3 | grep -Po '(?i)sas_token=\K([^,]*)')
+account_key=$(echo $3 | grep -Po '(?i)account_key=\K([^,]*)' || :)
+sas_token=$(echo $3 | grep -Po '(?i)sas_token=\K([^,]*)' || :)
 
 ps -C blobfuse --format "cmd" | [[ $(grep -c " $1 ") -eq 1 ]] && mountpoint -q $1 && exit 0
 [[ $(grep -c " $1 " /etc/mtab) -eq 1 ]] && { umount -f $1 || write_log "umount error code: $?"; }
 
-tmp_dir="/blobfusetmp/$account_name/$container_name"
+tmp_dir="/mnt/blobfusetmp/$account_name/$container_name"
+rm -rf $tmp_dir
```

Comment:
Making sure that the temp directory is empty (blobfuse requirement). -rf means recursive and don't prompt for confirmation.